packages/home-service/src/schema.ts

```diff
@@ -8,6 +8,10 @@ export const HomeServiceSchema: Schema = new Schema({
     entityType: String,
     colorScheme: String,
     videoUrl: String,
+    createdBy: String,
+    createdOn: Date,
+    modifiedBy: String,
+    updatedOn: Date,
```

Comment:
rename this to modifiedOn instead of updatedOn. for the consistency

Reply:
@diwanshi I think we should make it `updatedOn` and `updatedBy` in all the microservices.
This is more in line with standard practices used in other APIs.

Reply:
Updated from`modifiedBy` to `updatedBy`